streams/src/components/rtp/jpeg-headers.ts

```diff
@@ -173,5 +175,5 @@ export function makeScanHeader() {
 }
 
 export function makeDRIHeader(dri: number) {
-  return Buffer.from([0xff, 0xdd, 0x00, 4, dri >> 8, dri & 0xff])
+  return new Uint8Array([0xff, 0xdd, 0x00, 4, dri >> 8, dri & 0xff])
```

Comment:
Reading code like this while not knowing much about video parsing just makes me feel stupid. Am I supposed to know what theses hex codes mean or what `dri` is and what shifting it with 8 does?

To an "outsider", much of this code reads like I would expect minified and obfuscated code to look like. :laughing: 

Only one day away from the weekend... :beers:

Reply:
Much of this is legacy that I did not want to touch, so yes, legibility isn't that great in a lot of places.